examples/agent-sandbox-rl/tests/test_config.py

```
@@ -0,0 +1,80 @@
+  t = TemplateSpec(runtime_class="gvisor",
+                   node_selector={"cloud.google.com/gke-nodepool": "e2-pool"},
+                   image_pull_secret="dockerhub-pro")
+  cfg = FleetConfig(template=t, max_concurrent=8)
+  assert cfg.template.runtime_class == "gvisor"
+  assert cfg.template.node_selector["cloud.google.com/gke-nodepool"] == "e2-pool"
+
+
+def test_template_name_prefix_rejects_invalid():
+  import pytest
+  from agent_sandbox_rl import FleetConfig
+  with pytest.raises(Exception):           # pydantic ValidationError
```

Ruff (0.15.17)

[warning] Do not assert blind exception: `Exception`

(B017)